frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -1,14 +1,32 @@
 import React from 'react'
```

Comment:
Bunch of TS errors reported in this file. Possibly due to changes that got merged in with https://github.com/PostHog/posthog/pull/4839

Reply:
ah you're right I think generalizing `ActionFilter` to `ActionFilter | 'sessions'` broke some things